include/mbedtls/mbedtls_config.h

```diff
@@ -256,6 +256,18 @@
  */
 //#define MBEDTLS_DEPRECATED_REMOVED
 
+/** \def MBEDTLS_CHECK_RETURN
+ *
+ * This macro appearing at the beginning of the declaration of a function
+ * indicates that its return value should be checked.
+ *
+ * Default implementation resides in platform_util.h.
+ * You can override default implementation by defining your own.
```

Comment:
Minor: I think we need an article here - _the_ default implementation - and on the line above. (And also below: the custom implementation.)